README.md

```diff
@@ -55,7 +55,7 @@ Note: the levels on codecombat.com are [not open source](LICENSE-LEVELS.md).
 ![George Saines](https://s3.amazonaws.com/files.codecombat.com/wiki-images/avatars/George%20Saines/george_small.png "George Saines")
 ![Scott Erickson](https://s3.amazonaws.com/files.codecombat.com/wiki-images/avatars/Scott%20Erickson/scott-avatar.png "Scott Erickson")
 ![Matt Lott](https://s3.amazonaws.com/files.codecombat.com/wiki-images/avatars/Matt%20Lott/matt-avatar.png "Matt Lott")
-![Catherine Weresow](https://s3.amazonaws.com/files.codecombat.com/wiki-images/avatars/Cat%20Weresow/cat-avatar.png "Catherine Weresow")
+![Catherine Weresow](https://sjc1.discourse-cdn.com/business6/user_avatar/discourse.codecombat.com/catsync/100/6431_2.png "Catherine Weresow")
```

Comment:
Thank you so much for fixing this.
I copied your file to our s3 bucket so that we can host the avatar.

Could you update the url to: https://s3.amazonaws.com/files.codecombat.com/wiki-images/avatars/Cat+Weresow/cat_sync.png

Thank you!

```suggestion
![Catherine Weresow](https://s3.amazonaws.com/files.codecombat.com/wiki-images/avatars/Cat+Weresow/cat_sync.png "Catherine Weresow")
```

Reply:
Thanks. I've changed the link to the hosted one. Thanks Andrew!